.github/CONTRIBUTING.md

```diff
@@ -142,7 +149,10 @@ builder failures, you may be experiencing a [test suite
 issue](https://github.com/zfsonlinux/zfs/issues?q=is%3Aissue+is%3Aopen+label%3A%22Test+Suite%22).
 There are also various [buildbot options](https://github.com/zfsonlinux/zfs/wiki/Buildbot-Options)
 to control how changes are tested.
-* All proposed changes must be approved by a ZFS on Linux organization member.
+* If your pull request has not yet passed the test suite,
+please think about submitting it as a draft PR instead.
+* Tests include the following and more:
+`zfstest`, `ztest`/`zloop`, `make cppcheck` and `make cstyle`
```

Comment:
Let's include a link here to some of the wiki documentation which describes how to run `zloop.sh` and `zfs-tests.sh` locally.   https://github.com/zfsonlinux/zfs/wiki/Building-ZFS#running-zloopsh-and-zfs-testssh

I'd suggest we add a separate bullet illustrating that `make cppcheck` and `make cstyle` can be run locally.

Reply:
I think the wiki (for example the same page as zloop and zfstest) is a beter place to explain how to run cppcheck and cstyle.

I'll point cppcheck to the readme of cppcheck for now, link can always be changed